solr/solrj/src/java/org/apache/solr/common/util/Utils.java

```diff
@@ -712,7 +712,7 @@ public static String getBaseUrlForNodeName(
     }
     final String hostAndPort = nodeName.substring(0, _offset);
     final String path = URLDecoder.decode(nodeName.substring(1 + _offset), UTF_8);
```

Comment:
"path" isn't needed now?  Or just leave this code as it was?

Reply:
Good catch.  So, at one time we needed the path variable to decide if you had `/solr/mysubsolr` etc..  but now, we have just `/solr` so this can be eliminated actually!